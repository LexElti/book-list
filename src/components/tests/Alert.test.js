import React from 'react'
import Alert from '../Alert'

describe('<Alert />', () => {
  const props = {
    text: 'Some message'
  }

  const wrapper = shallow(<Alert {...props} />)

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('correct render data', () => {
    expect(wrapper.find('.alert.alert-warning').text()).toEqual(props.text)
  })
})
