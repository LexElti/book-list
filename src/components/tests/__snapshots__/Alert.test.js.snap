// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert /> renders properly 1`] = `ShallowWrapper {}`;
